services/minespace-web/src/components/dashboard/mine/projects/NoticeOfWorkTable.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { FC } from "react";
+import { Link } from "react-router-dom";
+import { Row, Col, Badge } from "antd";
+import { formatDate } from "@/utils/helpers";
+import CoreTable from "@mds/common/components/common/CoreTable";
+import * as Strings from "@mds/common/constants/strings";
+import { INoticeOfWork } from "@mds/common/interfaces";
+import DocumentLink from "@mds/common/components/documents/DocumentLink";
+import { downloadNowDocument } from "@common/utils/actionlessNetworkCalls";
+import { isEmpty } from "lodash";
+import { getApplicationStatusType } from "@mds/common/constants/badgeStatusTypes";
+import { SortOrder } from "antd/es/table/interface";
+import { dateSorter } from "@mds/common/redux/utils/helpers";
+
+interface NoticeOfWorkTableProps {
+  isLoaded: boolean;
+  applications: INoticeOfWork[];
+}
+
+const transformRowData = (applications: INoticeOfWork[]) =>
+  applications?.map((application) => ({
+    key: application.now_application_guid,
+    now_application_guid: application.now_application_guid,
+    now_number: application.now_number || Strings.EMPTY_FIELD,
+    mine_name: application.mine_name || Strings.EMPTY_FIELD,
+    mine_guid: application.mine_guid,
+    notice_of_work_type_description:
+      application.notice_of_work_type_description || Strings.EMPTY_FIELD,
+    lead_inspector_name: application.lead_inspector_name || Strings.EMPTY_FIELD,
```

Comment:
looks like there's a lot of fields here not used in the table!
I prefer a null coalesce over a OR operator, but I'll also note that the renderXColumn functions in CoreTableCommonColumns will automatically  do that (as well as having date & string sorters)